scripts/macos/RenameMacUserNameAndHomeDirectory.sh

```diff
@@ -1,13 +1,21 @@
 #!/bin/bash
```

Comment:
We should add the `e` and `u` flags here? For example: `#!/bin/bash -eu`

That would cause the script to exit on any command failures and complain about unused variables. It helps make Bash act more like a "real" language.